Assets/Scripts/PlayerShooting.cs

```diff
@@ -4,25 +4,43 @@
 
 public class PlayerShooting : MonoBehaviour
 {
+    
     public GameObject arrowPrefab;
-    public float speed = 100f;
+    
+    public static float minForce = 0f;
```

Comment:
im not sure why all of these are static. Static variables don't show up in the inspector so you cant change them in the editor. being able to adjust variables in the editor is very useful